test/integration/multi_arg.toml

```diff
@@ -20,7 +20,7 @@ definitions = '''
 
   # Test that TreeBuilder failing to run on the first arg doesn't impact the second arg
   [cases.tb_fail_first_arg]
-    oil_skip = "oil doesn't handle invalid strings"
+    oil_disable = "treebuilder tests are oid specific"
```

Comment:
I think this description is a little misleading. Tried running it and OIL crashes with an exception for this test.

It's more like OIL can't handle invalid data safely, similar to not chasing raw pointers safely, rather than this test not applying to it.

Reply:
Another thought: could we wrap OIL's code in a giant try/catch block to handle errors like this?

Reply:
> I think this description is a little misleading. Tried running it and OIL crashes with an exception for this test.
> 
> It's more like OIL can't handle invalid data safely, similar to not chasing raw pointers safely, rather than this test not applying to it.

I think this test is still invalid for oil. I shoehorned oil in for the multi arg tests for compatibility reasons, but the way it runs is:
```
size_t size0 = 0;
auto ret0 = ObjectIntrospection::getObjectSize(&a0, &size0, opts);
std::cout << "{\"ret\": " << ret0 << ", \"size\": " << size0 << "}" << std::endl;
std::cout << "," << std::endl;
size_t size1 = 0;
auto ret1 = ObjectIntrospection::getObjectSize(&a1, &size1, opts);
std::cout << "{\"ret\": " << ret1 << ", \"size\": " << size1 << "}" << std::endl;
std::cout << "]}" << std::endl;
```

As currently, unlike oid, oil has no concept of multi arg probing (except perhaps shoving references in a tuple, but we can't yet do that either). I don't think this test in its current form makes sense for oil, we could instead write different ones to check it doesn't crash.

> Another thought: could we wrap OIL's code in a giant try/catch block to handle errors like this?

This is a test framework error and not an oil error. It tries to add up the static and dynamic size of the `expect_json` to get an oil size, but the expect json for it looks like this:

`expect_json = '[{},{"staticSize":12, "dynamicSize":0}]'`

With no `staticSize` node in the first object. This could be fixed to default to 0 or something, but I think this test is invalid for oil anyway.